contrib/opencensus-ext-azure/opencensus/ext/azure/trace_exporter/__init__.py

```diff
@@ -69,18 +69,24 @@ def span_data_to_envelope(self, sd):
                     sd.start_time,
                     sd.end_time,
                 ),
-                responseCode='0',  # TODO
-                success=True,  # TODO
+                responseCode='0',
+                success=False,
                 properties={},
             )
             envelope.data = Data(baseData=data, baseType='RequestData')
             if 'http.method' in sd.attributes:
                 data.name = sd.attributes['http.method']
+            if 'http.route' in sd.attributes:
+                data.name = data.name + ' ' + sd.attributes['http.route']
+                envelope.tags['ai.operation.name'] = data.name
             if 'http.url' in sd.attributes:
-                data.name = data.name + ' ' + sd.attributes['http.url']
                 data.url = sd.attributes['http.url']
             if 'http.status_code' in sd.attributes:
-                data.responseCode = str(sd.attributes['http.status_code'])
+                status_code = sd.attributes['http.status_code']
+                data.responseCode = str(status_code)
+                data.success = (
+                    status_code >= 200 and status_code <= 299
```

Comment:
Just FYI - we were having a debate whether HTTP 3xx is considered as success or not.
I think it's fine to keep [200, 300) for this PR.

Reply:
Fixed!!